nav2_msgs/msg/CostmapFilterInfo.msg

```diff
@@ -0,0 +1,11 @@
+std_msgs/Header header
+# Type of plugin used (keepout filter, speed limit in m/s, speed limit in percent, etc...)
+uint8 type
+# Name of map filter topic
+string map_filter_topic
```

Comment:
Same here with the 'map filter' name. That's not descriptive or even really consistent with the naming we've discussed. The filter is the plugin, not the map mask for it.

Reply:
Renamed in there: https://github.com/ros-planning/navigation2/commit/cfeeb1a0b96560bb68fa8f8119e27a62c65e7100#diff-ea61287bc0126908adac952fac129bc2L4